src/services/utils/jwt.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
+import jwt from 'jsonwebtoken'
+import { makeLogger } from 'lido-nanolib'
+
+export type JwtService = ReturnType<typeof makeJwtService>
+
+export const makeJwtService = (
+  logger: ReturnType<typeof makeLogger>,
+  { JWT_SECRET_PATH }: { JWT_SECRET_PATH: string }
+) => {
+  const generateToken = () => {
+    try {
+      // read the secret from the file
+      const hexSecret = fs.readFileSync(JWT_SECRET_PATH, 'utf8').trim()
+      // convert the hex to a buffer
+      const secretBuffer = Buffer.from(hexSecret, 'hex')
```

Comment:
It seems that we’re generating a new token for every request? Perhaps this logic should be moved to the configuration, so the token is loaded once at application startup and validated there? It would also be great to have a test for this.

Reply:
good suggestion, improvement completed